nginx-regex-tester/regextester/Dockerfile

```
@@ -5,7 +5,7 @@ RUN chmod +x /usr/local/sbin/start.sh
 
 RUN apt-get update && apt-get install -y -q wget curl apt-transport-https lsb-release ca-certificates gnupg
 
-RUN wget -q -O - http://nginx.org/keys/nginx_signing.key | apt-key add -
+RUN wget -q -O - http://nginx.org/keys/nginx_signing.key | gpg --dearmor | tee /usr/share/keyrings/nginx-archive-keyring.gpg
```

You don't actually need to use `tee` since the output is not human readable (and it's hidden by docker by default)
```suggestion
RUN wget -q -O - http://nginx.org/keys/nginx_signing.key | gpg --dearmor > /usr/share/keyrings/nginx-archive-keyring.gpg
```